db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20210310000709) do
+ActiveRecord::Schema.define(version: 20210413044542) do
```

Comment:
No worries 👍 I'll add some API tests as well 😇

Also, 6354affa6162a474c7caf6fb2ee769e204727340 was because the migration [`20210413044542_change_do_not_resubmit.rb`](https://github.com/doubtfire-lms/doubtfire-api/blob/d34de6c25429fd62d0aa36a31d3f5732700894bf/db/migrate/20210413044542_change_do_not_resubmit.rb) was _added, but not applied to the schema_, causing [`rake db:populate` in `setup.sh`](https://github.com/doubtfire-lms/doubtfire-api/blob/d34de6c25429fd62d0aa36a31d3f5732700894bf/setup.sh#L234) to fail.

I could revert it, but that would mean `bundle exec rake db:migrate` must be manually executed during setup.
Thoughts? 🤔